Updated analyze within api.py

HRicketts007 opened the pull request.

HRicketts007 committed:
updated analyze in api

liamellison02 commented on api/flaskapp/twitter_oauth.py:
Make sure to not commit load_dotenv b/c that package is built to load local env variables from an env file

liamellison02 commented on api/.env:
Also please delete this immediately, our GitHub repo is public and you just shared our secret keys with anyone who views this branch

HRicketts007 committed:
Delete api/.env

HRicketts007 committed:
updated process.py to train model

Please leave tensorflow prefix in the imports it will not work otherwise. Ik it has an error but it works anyway.
